vendor/brightray/browser/linux/libnotify/notification.h

```diff
@@ -0,0 +1,190 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 8 -*-
```

Comment:
where did these files come from? Can we pull them in from chromium src?

Reply:
I copied the files from my distribution package. Chromium is not using `libnotify` but calling dbus directly.